src/toSVG.js

```diff
@@ -272,7 +272,13 @@ export default (parsed) => {
         acc.bbox.expandByPoint(bbox.min)
         acc.bbox.expandByPoint(bbox.max)
       }
-      acc.elements.push(`<g stroke="${rgbToColorAttribute(rgb)}">${element}</g>`)
+      let strokeDashArray = ''
+
+      if (entity.lineTypeName && entity.lineTypeName.toLowerCase() !== 'continuous') {
```

Comment:
Seems like this code assumes that continuous and dashed are the only options, is that correct?